src/models/usuario.js

```diff
@@ -1,13 +1,14 @@
 const { Schema, model } = require('mongoose');
+const bcryptjs = require('bcryptjs')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7127635)
```suggestion
const bcryptjs = require("bcryptjs")
```